docker/Dockerfile-ubuntu-20

```diff
@@ -10,7 +10,14 @@ RUN apt update && \
         ninja-build \
         python3-pip \
         python3-venv \
-        python3-dev curl gnupg apt-transport-https && \
+        python3-dev curl gnupg apt-transport-https \
+        zlib1g \
+        wget && \
+    wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
+    dpkg -i packages-microsoft-prod.deb && \
+    rm packages-microsoft-prod.deb && \
+    apt update && \
```

Comment:
Do we really need it here? You used it because of microsoft packages?

Reply:
It's the only way i managed to make dotnet work properly, but there should be a better way
That's why it is the only docker file i have changed for now